config/environments/production.rb

```diff
@@ -94,26 +94,24 @@
   ]
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
-  # config.log_formatter = ::Logger::Formatter.new
   config.log_formatter = ::Logger::Formatter.new
 
-  # Use a different logger for distributed setups.
-  # require 'syslog/logger'
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
-
   if ENV["RAILS_LOG_TO_STDOUT"].present?
+    # Use a different logger for distributed setups.
+    # require 'syslog/logger'
+    # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
+
     logger           = ActiveSupport::Logger.new(STDOUT)
     logger.formatter = config.log_formatter
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
+  elsif (ENV["SEND_LOGS_TO_TIMBER"] || "true") == "true"
```

Comment:
**question:** Won't this always end up equating to true with `|| "true" == "true"`?

Reply:
JK, is it Friday yet? 😅 

That would only be the case if `ENV["SEND_LOGS_TO_TIMBER"]` were `nil`.

Reply:
LOL This comment just made my day....btw I am right there with you https://github.com/thepracticaldev/dev.to/pull/9204#discussion_r452302134 😅

Reply:
LOL.

Do this mean the scenario where `RAILS_LOG_TO_STDOUT` isn't set (skips first conditional of `if`) and `SEND_LOGS_TO_TIMBER` is `"false"` (skips second conditional of `elsif`) should never happen? In that case, neither of these conditionals would be hit and we don't have an `else` here.

Reply:
That should never happen but I think Rails might default to STDOUT in that case. This little case statement will be removed as soon as we make the switch and confirm everything is working

Reply:
Ahhhh I see - thank you!